server/server.js

```diff
@@ -7,14 +7,14 @@ process.on('uncaughtException', (err) => {
   process.exit(1);
 });
 
-dotenv.config({ path: './config.env' });
+dotenv.config({ path: '.env' });
 const app = require('./app');
 
 const DB = process.env.DATABASE;
 
 mongoose.connect(DB).then(() => console.log('DB Connection Successful!'));
 
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 3500;

```

Comment:
Its not a good idea to change the PORT number, even if you do it while testing please revert the PORT numbers back to the original ones